band_together/scenes/levels/level_0.tscn

```diff
@@ -1,11 +1,11 @@
-[gd_scene load_steps=13 format=4 uid="uid://bnpo75dm0wp2n"]
+[gd_scene load_steps=12 format=4 uid="uid://bnpo75dm0wp2n"]
 
 [ext_resource type="PackedScene" uid="uid://f860pnqtg4hh" path="res://scenes/levels/parallax.tscn" id="1_1gg8b"]
 [ext_resource type="Texture2D" uid="uid://7nmwi83e07ao" path="res://Environment/beach_tileset.png" id="1_oparx"]
 [ext_resource type="Texture2D" uid="uid://da6a4a38e1tul" path="res://Environment/Tiles-DELETE.png" id="2_oi3pf"]
 [ext_resource type="PackedScene" uid="uid://chmdbpxvcc4i8" path="res://entities/player/player.tscn" id="3_wdqm4"]
 [ext_resource type="PackedScene" uid="uid://bvur20x1mukp3" path="res://entities/enemies/rocks/rock.tscn" id="4_mfn4g"]
-[ext_resource type="Script" path="res://scenes/test/fall_area.gd" id="5_m44e5"]
+[ext_resource type="PackedScene" uid="uid://d1corgwhnfoyh" path="res://scenes/world_boundary.tscn" id="6_nuwux"]
```

Comment:
Made the world boundary a scene in the hopes we can use it to calculate the world boundaries in other scenes... But when I couldn't get it to work, wasn't sure how to undo it quickly. Keeping it here because maybe I'll try again in the future!